### DISCUSSION TITLE
Relax Mergeability requirements

### DISCUSSION THREAD

**Discussion by jalvz** [elastic/e2e-testing#1050] [General]
Is it possible to relax merge-ability requirements for this project? 

I have been trying to merge a PR for a while, but every time I come to it I find that something has been merged since last time I checked, I have to merge master upstream, and wait until tests finish again, even if I know that there has no been related changes.

**Comment by mdelapenya** [comment]
@jalvz we are leveraging Github discussions since today, and this one seems a very valid discussion.

As we talked past week, we relaxed the rules so that you can autonomously merge (you can check the rules here: https://github.com/elastic/e2e-testing/settings/access)
